bridges/KleinanzeigenBridge.php

```diff
@@ -138,22 +136,56 @@ private function addItem($element)
     {
         $item = [];
 
+        $item['content'] = '';
+
+        $json = $element->find('.aditem-image > script', 0);
+        if ($json) {
+            $data = json_decode($json->innertext, true);
+            $item['title'] = $data['title'];
+            $item['content'] .= '<div><p>' . $data['description'] . '</div></p></br>';
+        } else {
+            $item['title'] = $element->find('h2', 0)->plaintext;
+            $item['content'] .= $element->find('.aditem-main--middle--description');
+        }
+
+        if ($element->find('.aditem-main--top', 0)) {
+            $item['content'] .= $element->find('.aditem-main--top', 0);
+        }
+
+        if ($element->find('.aditem-main--middle--price-shipping', 0)) {
+            $item['content'] .= preg_replace(
+                '#(<p\s+class="aditem-main--middle--price-shipping--old-price"[^>]*>.*?</p>)#si',
+                '<s>$1</s>',
+                $element->find('.aditem-main--middle--price-shipping', 0)
+            );
+        }
+
+        if ($element->find('.aditem-main--bottom', 0)) {
+            $item['content'] .= $element->find('.aditem-main--bottom', 0);
+        }
+
+        $item['content'] = sanitize($item['content']);
+
         $item['uid'] = $element->getAttribute('data-adid');
-        $item['uri'] = $element->getAttribute('data-href');
-
-        $item['title'] = $element->find('h2', 0)->plaintext;
-        $item['timestamp'] = $element->find('div.aditem-main--top--right', 0)->plaintext;
-        $imgUrl = str_replace(
-            'rule=$_2.JPG',
-            'rule=$_57.JPG',
-            str_replace(
-                'rule=$_35.JPG',
-                'rule=$_57.JPG',
-                $element->find('img', 0) ? $element->find('img', 0)->getAttribute('src') : ''
-            )
-        ); //enhance img quality
-
-        $item['content'] = '<img src="' . $imgUrl . '"/>' . $element->find('div.aditem-main', 0)->outertext;
```

Comment:
I would still add the image into the content HTML in addition to the enclosure. Depending on RSS reader, the display of this stuff can vary widely.
If you want, you can add a bridge option to control that.